taxcalc/params.json

```diff
@@ -394,9 +394,9 @@
     },
      "_EITC_rt":{
         "long_name": "Earned income credit rate",
-	    "description": "Apply this rate on Income.",
+	"description": "If one taxpayer's AGI is below AGI phaseout start, he or she can apply this rate to your earned income to calculate the credit amount eligible. ",
```

Comment:
Maybe -> "If one taxpayer's AGI is below AGI phaseout start, he can apply this rate to his earned income to calculate the credit amount eligible. "